.github/workflows/composite-actions/upload-artifact/action.yaml

```diff
@@ -0,0 +1,32 @@
+name: "Upload artifact"
+
+inputs:
+  path:
+    description: "Path for files to include"
+    required: true
+  name:
```

Comment:
I see that in some calls to this job (`performance-dump-attempt-${{ github.run_attempt }}` for example) the run attempt is included in the name.

I think it would be useful to callout in the description or a comment, whether callers are expected to supply the run_attempt, or whether the composite-action will handle that individually.

Reply:
It doesn't matter at all what the user supplies, the name will still be unique
If they want to include attempt number, then great, if they don't, also great
To the extent that including it is an effort to ensure uniqueness, it's no longer required and can be removed (see comment [here](https://github.com/solo-io/gloo/pull/10743/files#r2027538827))

Reply:
My 2 cents is that anything that is under `${{ github.* }}` should be handled by the composite action and the engineer shouldn't have to worry about it. But, i don't think it's a major issue.

Reply:
The action should handle all the variables that will be uniform across runs and only the parts of the name that is unique to the artifact and context should be the engineer's concern.